cli/command/container/cp.go

```diff
@@ -34,11 +39,70 @@ const (
 type cpConfig struct {
 	followLink bool
 	copyUIDGID bool
+	quiet      bool
 	sourcePath string
 	destPath   string
 	container  string
 }
 
+//copyReader is a Wrapper to content
+type copyReader struct {
```

Comment:
Maybe `cpWithProgress`, and a doc string that describes the usage.

Reply:
Ok, I'll update this.